vowpalwabbit/core/include/vw/core/confidence_sequence_robust.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#pragma once
+
+#if !defined(__APPLE__) && !defined(_WIN32)
+#  define __STDCPP_MATH_SPEC_FUNCS__ 201003L
+#  define __STDCPP_WANT_MATH_SPEC_FUNCS__ 1
+#endif
+
+#include "vw/core/io_buf.h"
+#include "vw/core/metric_sink.h"
+
+namespace VW
+{
+class g_tilde
+{
+public:
+  g_tilde(double k);
+  double histo_variance(std::map<std::pair<uint64_t, bool>, double>& hist, double lam_sqrt_tp1);
+  void histo_insert(std::map<std::pair<uint64_t, bool>, double>& hist, double x);
+  void add_obs(double x);
+  double get_s();
+  double get_v(double lam_sqrt_tp1);
+
+private:
+  double k;
+  double log_k;
+  double sum_x;
+  double sum_low_v;
+  double sum_mid_v;
+  std::map<std::pair<uint64_t, bool>, double> sum_v_histo;
+  uint64_t t;
+};
+
+class countable_discrete_base
```

Comment:
what is the reason behind this name?

Reply:
this is based on some of Paul's naming conventions

Reply:
maybe add a comment referencing on top of this class or whichever is more relevant to that other repo with the robust impl of Paul

Reply:
added comments connecting the code